src/modules/popullateMovie.js

```diff
@@ -4,84 +4,105 @@ import Involve from './involvementApi.js';
+    </svg><b>${like.likes}</b> likes</span>
+    </div>
+    <div class="btn-container">
+    <button onclick="popUp(${res.id},'${name}')" class="btn like" id="${name}">Comments</button>
+    <button class="btn">Reservations</button></di>
+    </div>`;
+    count();
+    document.querySelectorAll('.like').forEach((like) => {
+        like.addEventListener('click', async (e) => {
+            // call update function
+        await updateLikes(e.target.parentNode)
+        
+        });
+        });
+        
+    document.querySelectorAll('.card').forEach((card) => {
+        card.addEventListener('load',  count);
+        });
+});    
+})
+}
+
+window.popUp = async (id,name) => {
+let commentList = "";
+const comments = await Involve.getComments(name);
+    if(comments.length > 0) {
+        comments.forEach((com) => {
+        commentList += `<div> <p>${com.creation_date}<span>${cDay}/ ${cMonth}/ ${cYear}</span></p>
+        <p>${com.username}:</p>
+        <p>${com.comment}</p></div>`; 
+        return commentList;
+    });
+}
+commentList = commentList ? commentList : "Be the first to comment";
+
+pop.classList.add('visible'); 
 
-        document.querySelectorAll('.like').forEach((like) => {
-            like.addEventListener('click', async (e) => {
-              // call update function
-            await updateLikes(e.target.parentNode)
-            
-            });
-          });
-          
-        document.querySelectorAll('.card').forEach((card) => {
-            card.addEventListener('load',  count);
-          });
-    });    
-  })
-  
-    window.popUp = async (id) => {
-      console.log(id)
-     const pop = document.querySelector('.pop');
-     pop.classList.add('visible');
-     await Movies.displayMovie()
-     .then((res) => {
-        res.forEach(res => {
-           if(res.id === id){
-             pop.innerHTML = `
-            <div class="content">
-              <div class="popup-body">
-                  <h2 class="name">${res.name}</h2>
-                  
-                  <span class="close">&times;</span>                   
-              </div>
-              <div class="poster">
-                  <img src="${res.image.original}" alt="${res.name}">
-              </div>
-              <div class="movie-type">
-                 ${res.summary}
-                
-              </div>
-              <div class="movie-details">
-                
-                  <p class="runtime">Duration: ${res.runtime} mins</p>
-                  <p class="airdate">Airing date: ${res.airdate}</p>
-              </div>
-            </div>`;
-            
-            const closeBtn = document.querySelector('.close');
-            closeBtn.addEventListener('click', () => {
-                window.location.reload();
-                pop.classList.remove('visible');
-            });
-           }
-           
-            })
```

Comment:
What you did here is a great job. As a [Optional] suggestion kindly consider making your code modular. It will help your code to become robust, easily maintainable, and reusable. Additionally, it is a best practice, it will help you in your future career. 😉